istio.deps

```diff
@@ -4,7 +4,7 @@
 		"name": "PROXY_REPO_SHA",
 		"repoName": "proxy",
 		"file": "",
-		"lastStableSHA": "49bf92867731f6c108581e55131083fe0b35687f"
+		"lastStableSHA": "3245d3582d7e4937150b2c56120a556420721ac3"
```

Comment:
This should now be https://github.com/istio/proxy/commit/42eec5f2d1af7718491b98fa82afb14118d1fff3, from https://github.com/istio/proxy/pull/2384.